src/main/java/com/shaft/tools/internal/support/JavaHelper.java

```diff
@@ -97,7 +97,7 @@ public static String convertBase64(String text) {
      */
     public static int compareTwoObjects(Object expectedValue, Object actualValue, Object comparisonType,
                                         Boolean validationType) {
-        ReportManager.logDiscrete("Expected \"" + expectedValue + "\", and actual \"" + actualValue + "\"");
+        //ReportManager.logDiscrete("Expected \"" + expectedValue + "\", and actual \"" + actualValue + "\"");
```

Comment:
instead of commenting this line, kindly log it using DEBUG level.

Reply:
I tried to logging using the `DEBUG` level but to no avail. As shown in the screenshot below, The message is still logged to the console as an `INFO`.

![Screenshot](https://github.com/ShaftHQ/SHAFT_ENGINE/assets/37153097/28076dd3-b1df-4d76-bce1-8aec8d0764d9)